app/src/components/CountdownTimer.jsx

```diff
@@ -35,18 +34,16 @@ function CountdownTimer({ deadline }) {
 
         timerComponents.push(
             <span>
-                {timeLeft[interval]} {interval}{" "}
-            </span>
+                {timeLeft[interval]} {interval}{' '}
+            </span>,
```

Comment:
Is it extra `,`?

Reply:
Yes, this is from the prettier code format. This itself is not too much of an issue but the Countdown component will require several refactoring later